plugins/engines/ospray/OSPRayEngine.cpp

```diff
@@ -198,6 +198,7 @@ void OSPRayEngine::preRender()
     const auto& renderParams = _parametersManager.getRenderingParameters();
     if (renderParams.getAccumulation() != _frameBuffer->getAccumulation())
         _frameBuffer->setAccumulation(renderParams.getAccumulation());
+    _camera->takeSceneParameters(*_scene);
```

Comment:
just cast camera to OSPRayCamera to not need takeSceneParameters() in base class

Reply:
I don't even need to cast because _camera is an OSPCamera. Originally I had _camera->setClipPlanes(_scene->getClipPlanes()), but I changed it to make the smallest modification that also works in the SnapshotFunctor.